src/nwb_datajoint/figurl_views/SpikeSortingRecordingView.py

```diff
@@ -0,0 +1,95 @@
+from typing import List, Union
+import datajoint as dj
+import sortingview as sv
+from sortingview.SpikeSortingView import create_raw_traces_plot
+from sortingview.SpikeSortingView.Figure import Figure
+import numpy as np
+import kachery_client as kc
+import os
+import spikeinterface as si
+
+from ..common.common_spikesorting import SpikeSortingRecording
+
+schema = dj.schema('figurl_view_spike_sorting_recording')
+
+@schema
+class SpikeSortingRecordingView(dj.Computed):
+    definition = """
+    # Schema for storing figurl views of spike sorting recordings
+    -> SpikeSortingRecording
+    ---
+    figurl: varchar(10000)
+    """
+    def make(self, key):
+        # Get the SpikeSortingRecording row
+        a = (SpikeSortingRecording & key).fetch1()
```

Comment:
Would prefer a more descriptive variable name that is not just `a`

Reply:
fixed this